homework.md

```diff
@@ -44,37 +44,37 @@ Using CSS, how would I select every `<h1>` tag on the page?
 
 What is JavaScript?
 
-A programming language used on the internet
+JavaScript is the Programming Language for the Web. JavaScript can update and change both HTML and CSS
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+saving files on the cloud, working on a group , receiving feedback and reviews
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+You copy it
```

Comment:
I would also add:
You make your own copy of that repository, from another account Github into yours. The point here is to be able to implement your changes freely, on your copy, without affecting the original repository.